frontend/src/Components/LoginPage.tsx

```diff
@@ -79,7 +80,13 @@ const styling = css`
 const ProtoLoginPage = ({ socket }: { socket: WebSocketApi }) => {
   const firstInputRef: React.RefObject<HTMLInputElement> = React.useRef(null);
   const [user, setUser] = React.useState('');
-  const [session, setSession] = React.useState('');
+  const path = window.location.pathname.slice(1);
+  const sessionId = path.match(
```

Comment:
This generates a new ID whenever anything changes, including typing in the username field

Reply:
fixed